### PR TITLE
chore: add documentation changeset

### PR DESCRIPTION
**Pull request by FrankEssenberger**
## What Has Changed?

add documentation on function/action imports in batch requests. The logic is for the v2/v4 is:

retrieve request: 
- function import GET in v2
- function import v4

changeset request:
- function import non GET in v2
- action import v4

### CODE REVIEW

File: docs-js/features/odata/odata-v2-client.mdx [hunk 1]
```diff
@@ -198,6 +198,10 @@ More advanced filter expressions can be found [here](#available-filter-expressio
 
 ## Batch Requests
 
+A retrieve request represents an HTTP `GET` request.
+In terms of the SAP Cloud SDK this includes all requests built by [`GetAllRequestBuilder`](#getall-request-builder), [`GetByKeyRequestBuilder`](#getbykey-request-builder) and [`FunctionImportRequestBuilder`]#(function-imports) using GET requests.
```
[the remark this answers is not in the document]
Comment (deekshas8) [reply docs-js/features/odata/odata-v2-client.mdx:202]: [req] This reads to me like an introduction to the "Retrieve Requests" section and should perhaps be moved there.

File: docs-js/features/odata/common/batch/responses.mdx [hunk 2]
```diff
@@ -51,6 +51,23 @@ async function getAddressesByIds(
 }
 ```
 
+Function imports using `GET` operation are part of the retrieve requests as well.
+As function imports are not entity based you obtain the response using the `responseTransformer` which is part of the `FunctionImportRequestBuilder`:
```
Comment (deekshas8) [inline docs-js/features/odata/common/batch/responses.mdx:55]:
```suggestion
As function imports are not entity based, you obtain the response using the `responseTransformer`, which is part of the `FunctionImportRequestBuilder`:
```

File: docs-js/features/odata/common/batch/responses.mdx [hunk 3]
```diff
@@ -100,6 +116,8 @@ async function updateAddresses(
 }
 ```
 
+As discussed for the [read response](#ReadResponse) you have to use the `responseTransformer` method for write responses related to function or action imports.
```
Comment (deekshas8) [inline docs-js/features/odata/common/batch/responses.mdx:119]:
```suggestion
As discussed for the [read response](#readresponse) you have to use the `responseTransformer` method for write responses related to function or action imports.
```

[req] The navigation does not work with #ReadResponse

File: docs-js/features/odata/odata-v2-client.mdx (hunk 1 above)
[the remark this answers is not in the document]
Comment (FrankEssenberger) [reply docs-js/features/odata/odata-v2-client.mdx:202]: yes position is wrong - moved to retrieve request subsection beginning.

File: docs-js/features/odata/odata-v4-client.mdx [hunk 4]
```diff
@@ -261,7 +265,10 @@ Note that we currently do not support the usage of count within a filter conditi
 
 ### Changeset
 
+A changeset is a collection of HTTP `POST`, `PUT`, `PATCH` and `DELETE` operations - requests built by [`CreateRequestBuilder`s](#create-request-builder), [`UpdateRequestBuilder`s](#update-request-builder), [`DeleteRequestBuilder`s](#delete-request-builder) or [`ActionImportRequestBuilder`](#action-imports) in terms of the SAP Cloud SDK.
```
[the remark this answers is not in the document]
Comment (FrankEssenberger) [reply docs-js/features/odata/odata-v4-client.mdx:268]: I head the same impression, but thought it was already there. But if a second pair of eyes sees the same issue I change it. I also split the sentence in two.

**Review by deekshas8 — APPROVED** [review]
Looks good!

**Pull request merged**